### PR TITLE
fix!: updated chain assets, removed `beta-5` network

### LINKED ISSUE #2281: Incorrect asset for devnet
We export [assets](https://github.com/FuelLabs/fuels-ts/blob/c880c5ee1e5a162a2a5ed1e3f4c336abedd278b4/packages/account/src/providers/assets/index.ts#L5) that can be easily consumed by the end user.

- The [`assetId`](https://github.com/FuelLabs/fuels-ts/blob/c880c5ee1e5a162a2a5ed1e3f4c336abedd278b4/packages/account/src/providers/assets/index.ts#L31) is currently `0x0000….` (or baseAssetId), is incorrect for devnet and testnet.
  - Expected: `0xf8f8b6283d7fa5b672b530cbb84fcccb4ff8dc40f8176ef4544ddb1f1952ad07`
 
- The [`chainId`](https://github.com/FuelLabs/fuels-ts/blob/c880c5ee1e5a162a2a5ed1e3f4c336abedd278b4/packages/account/src/providers/chains.ts#L8) for devnet should be 0.

> Reported by @LuizAsFight

### PR DESCRIPTION
**Pull request by petertonysmith94**
Closes #2281 

### Breaking change

- Chain ID entry for `beta5` has been removed, suggest to use `testnet`.
  ```ts
  // Before
  import { CHAIN_IDS } from 'fuels';
  const beta5 = CHAIN_IDS.fuel.beta5;
  
  // After
  import { CHAIN_IDS } from 'fuels';
  const testnet = CHAIN_IDS.fuel.testnet;
  ```

### CODE REVIEW

**Comment by fuel-service-user** [comment]
✨ A PR has been created under the `rc-2432` tag on `fuels-wallet` repo.
https://github.com/FuelLabs/fuels-wallet/pull/1329

File: packages/account/src/providers/chains.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export const CHAIN_IDS = {
     foundry: 31337,
   },
   fuel: {
-    beta5: 0,
-    devnet: 10,
+    devnet: 0,
+    testnet: 0,
```
Comment (arboleya) [inline packages/account/src/providers/chains.ts:8]: I'm sure I'm just out of context here, but what's the reasoning behind using identical chain IDs for `devnet` and `testnet`?

> cc @Torres-ssf @danielbate

Comment (Torres-ssf) [reply packages/account/src/providers/chains.ts:8]: @petertonysmith94 Why it is worth dropping the asset for the devnet?

**Comment by LuizAsFight** [comment]
instead of using a static `assets.json` , could we have a script that would run post-build ? something like

```
const devnetProvider = await Provider.create(DEVNET_URL);
const devnetChainId = provider.getChainId();
const devnetBaseAssetId = provider.getBaseAssetId();

const testnetProvider = await Provider.create(TESTNET_URL);
const testnetChainId = provider.getChainId();
const testnetBaseAssetId = provider.getBaseAssetId();

const assets = {
..... // builds the asset structure we have
};

fs.write('assets.json', assets);
```

**Comment by Torres-ssf** [comment]
> instead of using a static `assets.json` , could we have a script that would run post-build ? something like
> 
> ```
> const devnetProvider = await Provider.create(DEVNET_URL);
> const devnetChainId = provider.getChainId();
> const devnetBaseAssetId = provider.getBaseAssetId();
> 
> const testnetProvider = await Provider.create(TESTNET_URL);
> const testnetChainId = provider.getChainId();
> const testnetBaseAssetId = provider.getBaseAssetId();
> 
> const assets = {
> ..... // builds the asset structure we have
> };
> 
> fs.write('assets.json', assets);
> ```

This would indeed be a good solution for this problem

**Review by danielbate — APPROVED** [review]

**Review by arboleya — APPROVED** [review]

**Review by maschad — APPROVED** [review]

**Pull request merged**